lemarche/www/siaes/views.py

```diff
@@ -113,6 +114,9 @@ def get(self, request, *args, **kwargs):
             meta=extract_meta_from_request(self.request),
             session_id=request.COOKIES.get("sessionid", None),
         )
+        user = self.request.user
+        if user.kind == user.KIND_BUYER:
+            add_to_contact_list(self.request.user, "buyer_download")
```

Comment:
```suggestion
            add_to_contact_list(user, "buyer_download")
```

ou alors enlever la ligne `user = self.request.user` ?

Reply:
bien vu haha